types/mocha.d.ts

```diff
@@ -0,0 +1,214 @@
+// Type definitions for mocha 2.2.5
```

Comment:
Can we get these files from a package manager rather than including them in our source repo?  The should go hand-in-hand with our npm dependencies, no?